cg/meta/workflow/balsamic.py

```diff
@@ -398,8 +399,12 @@ def get_parsed_observation_file_paths(self, observations: list[str]) -> dict:
 
         return verified_observations
 
-    def get_verified_gens_file_paths(self, sex: Sex) -> dict[str, str] | None:
+    def get_verified_gens_file_paths(self, sex: Sex, panel_bed: str) -> dict[str, str] | None:
```

Comment:
this seems to always return a dict now
```suggestion
    def get_verified_gens_file_paths(self, sex: Sex, panel_bed: str) -> dict[str, str]:
```

Reply:
True! Nice catch thanks! 🙏